src/JsonSchema/Enum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace JsonSchema;
+
+class Enum extends \MabeEnum\Enum
+{
```

Comment:
Hmm, are you sure? Why would we assume that any other uses of Enum also have something to do with messages?